src/script/index.js

```
@@ -204,6 +204,12 @@ function initFooterToggles() {
         if (arrow) {
             arrow.classList.toggle('footer__nav-arrow--rotated', !isExpanded);
         }
+
+        if (isExpanded) {
+            panel.style.maxHeight = null;
+        } else {
+            panel.style.maxHeight = panel.scrollHeight + 'px';
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)